test/blocks/quiz-entry/quiz-entry.test.js

```diff
@@ -156,4 +156,147 @@ describe('Quiz Entry Component', () => {
     await new Promise((resolve) => setTimeout(resolve, 100));
     expect(continueButton.classList.contains('disabled')).to.be.false;
   });
+  it('should navigate the carousel using keyboard commands', async () => {
+    const options = document.querySelectorAll('.quiz-option');
+    const option = document.querySelector('.quiz-option');
+    option.click();
+    await new Promise((resolve) => setTimeout(resolve, 100));
+    const carousel = document.querySelector('.quiz-options-container');
+    const rightArrowEvent = new KeyboardEvent('keydown', { key: 'ArrowRight' });
+    const leftArrowEvent = new KeyboardEvent('keydown', { key: 'ArrowLeft' });
+    carousel.dispatchEvent(rightArrowEvent);
+    await new Promise((resolve) => setTimeout(resolve, 100));
+    carousel.dispatchEvent(leftArrowEvent);
```

Comment:
Nit: Same here, we can skip caching
```suggestion
  carousel.dispatchEvent(new KeyboardEvent('keydown', { key: 'ArrowRight' }));
  await new Promise((resolve) => setTimeout(resolve, 100));
  carousel.dispatchEvent(new KeyboardEvent('keydown', { key: 'ArrowLeft' }));
```